bin/parse-server

```diff
@@ -10,26 +10,28 @@ if (process.env.PARSE_SERVER_OPTIONS) {
 	options = JSON.parse(process.env.PARSE_SERVER_OPTIONS);
 
 } else {
-
-	options.databaseURI = process.env.PARSE_SERVER_DATABASE_URI;
-	options.cloud = process.env.PARSE_SERVER_CLOUD_CODE_MAIN;
-	options.collectionPrefix = process.env.PARSE_SERVER_COLLECTION_PREFIX;
+  options.app                        = {};
```

Comment:
This should be a separate PR probably. I wanted to separate the application configuration from the server configuration (see `args.app` in `index.js`), so that I could pass `args.app` to a constructor: `new ParseApp(args.app);`. Basically just namespacing the app options. This would likely change to an array when we implement multiple apps.